rollout/analysis.go

```diff
@@ -3,6 +3,7 @@ package rollout
 import (
 	"context"
 	"fmt"
+	"github.com/argoproj/argo-rollouts/utils/defaults"
```

Comment:
a space is missing

Reply:
Hi @huikang
Sorry, what space do you mean?

Reply:
I meant a new line before line 6 to pass the lint? Sorry about the confusion

Reply:
done